Marlin/Marlin_main.cpp

```diff
@@ -104,6 +104,7 @@
  * M1   - Same as M0
  * M17  - Enable/Power all stepper motors
  * M18  - Disable all stepper motors; same as M84
+ * M19  - Resume print from current (or given) Z height (disables all movements below the current Z position, a file must be selected to print after executing this M code)
```

Comment:
@thinkyhead
Correct, this way (using a separate M19 code) it would cover all cases.